llarp/service/session.cpp

```diff
@@ -27,7 +27,7 @@ namespace llarp
       const auto lastUsed = std::max(lastSend, lastRecv);
       if (lastUsed == 0s)
         return intro.IsExpired(now);
-      return now > lastUsed && (now - lastUsed > lifetime || intro.IsExpired(now));
+      return now > lastUsed && now - lastUsed > lifetime;
```

Comment:
probably keep the parentheses around the latter condition, just for ease of reading.

Reply:
fair